spaces/S000065/properties/P000056.md

```diff
@@ -1,11 +0,0 @@
----
```

Comment:
This property is no longer deducable it seems

Reply:
Thanks for catching that.  It's because I mistakenly set P64 (Baire) = false, instead of true.  Will fix.